projects/igniteui-angular/migrations/update-11_1_0/index.spec.ts

```diff
@@ -631,6 +631,25 @@ export class CsvExportComponent {
         );
     });
 
+    it('should replace onValueChange and onValueChanged with valueChange and valueChanged in igx-slider', async () => {
```

Comment:
If that pr is targeting master (12.0.x)v, we have to move those migration changes.